lib/modules/manager/types.ts

```diff
@@ -94,6 +95,7 @@ export interface LookupUpdate {
   pendingVersions?: string[];
   newVersion?: string;
   updateType?: UpdateType;
+  mergeConfidenceLevel?: MergeConfidence;
```

Comment:
```suggestion
  mergeConfidenceLevel?: MergeConfidence | undefined;
```
needed for new typescript validation, fix others too